### PR TITLE
test: re-enable schema tests for the v2 api

### PR DESCRIPTION
**Pull request by watson**
~~Depends on #584~~

### CODE REVIEW

**Comment by watson** [comment]
Just rebased with api-v2 and force pushed

**Comment by watson** [comment]
@Qard ready for review

File: test/integration/api-schema/download-json-schemas.sh [hunk 1]
```diff
@@ -38,9 +43,14 @@ FILES=( \
...
+mkdir -p \
+  ${schemadir}/errors \
+  ${schemadir}/transactions \
+  ${schemadir}/spans \
+  ${schemadir}/metrics \
+  ${schemadir}/sourcemaps
 
 for i in "${FILES[@]}"; do
-  download_schema https://raw.githubusercontent.com/elastic/apm-server/master/docs/spec/${i} ${schemadir}/${i}
+  download_schema https://raw.githubusercontent.com/elastic/apm-server/v2/docs/spec/${i} ${schemadir}/${i}
```
Comment (Qard) [inline test/integration/api-schema/download-json-schemas.sh]: Side note: we should probably go through and leave some comments to update branch links before release.

Comment (watson) [reply test/integration/api-schema/download-json-schemas.sh]: Good point. I have similar notes related to update documentation links in the [v2 tracking issue](https://github.com/elastic/apm-agent-nodejs/issues/471). I've added this as a task there as well

**Pull request merged**